### PR TITLE
Fixed object contain new line

### PR DESCRIPTION
**Pull request by HillLiu**

### CODE REVIEW

File: test/visitors/object_expression.test.ts [hunk 1]
```diff
@@ -176,6 +176,13 @@ pluginTester({
     },
     code: `const a = { b: { c: 1 } }`,
     output: `const a = JSON.parse('{"b":{"c":1}}')`
+  }, {
+    title: 'Test new line',
+    pluginOptions: {
+      minJSONStringSize: 0
+    },
+    code: `const a = { b: { c: "1${"\\n"}2" } }`,
```
Comment (nd-02110114) [inline test/visitors/object_expression.test.ts:184]: I think this is not a valid code.... What code do you want to convert...?

スクリーンショット 2020-04-06 15 26 19

**Comment by HillLiu** [comment]
@nd-02110114 

You should transpile es6 template string to js code first.
When I try write unit test, it also make my brain hurt, I also curious unit test why not just use pure JS. 

```
eval(`const a = { b: { c: "1${"\\n"}2" } }; console.log({a})`);
```

File: test/visitors/object_expression.test.ts (hunk 1 above)
Comment (HillLiu) [reply test/visitors/object_expression.test.ts:184]: it is es6 template string, need transpile to js code first.

```
eval(`const a = { b: { c: "1${"\\n"}2" } }; console.log({a})`);
```

**Comment by HillLiu** [comment]
\u2028 is  from babel-plugin-proposal-json-strings

https://babeljs.io/docs/en/babel-plugin-proposal-json-strings

**Comment by nd-02110114** [comment]
> My pr want resolve that if the original object string contain "\n", it will make code syntax error.

If you want to resolve the error in the case that  the original object string contain "\n",
you try to insert backslash before "\n". The example is below.

スクリーンショット 2020-04-06 23 14 10

So, I think `str.replace(/\n/g, '\\n')` is better compared with `value.replace(/\n/g, '\\u2028')`

**Comment by nd-02110114** [comment]
> When I try write unit test, it also make my brain hurt, I also curious unit test why not just use pure JS.

I felt the same way. Now, I rewrite test codes 👍

**Comment by HillLiu** [comment]
> > When I try write unit test, it also make my brain hurt, I also curious unit test why not just use pure JS.
> 
> I felt the same way. Now, I rewrite test codes 👍

Cool @nd-02110114 , then I'll close it now.

**Pull request closed without merging**